### DISCUSSION TITLE
How to use in Pocket Host?

### DISCUSSION THREAD

**Discussion by gwashark** [benallfree/pocketpages#4] [Q&A]
Hey, saw this in the official Pocket Host Discord Server and it is really cool and I want to try this out for myself. Is there any way I can import it into my Pocket Host Server? I see that it's still in development so maybe that's why it isn't in the docs yet.

**Comment by benallfree** [comment]
Yes I have it working with PocketHost. I haven't written the official docs yet though. I'll write some docs tonight and post a reply here.
